app/avo/resources/user_resource.rb

```diff
@@ -26,7 +26,7 @@ class UserResource < Avo::BaseResource
 
   field :email_reset, as: :boolean
   field :handle, as: :text
-  field :hide_email, as: :boolean
+  field :public_email, as: :boolean
```

Comment:
Hey @simi! I was working on a similar change. @segiddins astutely pointed out that this _may_ [break the old audits](https://github.com/rubygems/rubygems.org/pull/3847#discussion_r1241257409). Have you had a chance to investigate this as well? 

Based on my preliminary investigations today, it looks like the audits would indeed get mucked up. I wanted to check in to confirm whether you came to a similar conclusion.

If so, I'll dig into the Avo docs later this week to see if they have any guidance on how they handle renaming of fields.

Reply:
Hm, with further investigations, it appears the audits aren't broken after all. I'd still like to do a sanity check with you Josef to affirm this finding.

Reply:
column removal will be handled in following PR (not still clear in which way), marking this as resolved